acstools/acs_destripe_plus.py

```diff
@@ -607,6 +607,15 @@ def crrej_plus(filelist, outroot, keep_intermediate_files=False, verbose=True):
     if os.path.isfile(wrongname) and not os.path.exists(rightname):
         os.rename(wrongname, rightname)
 
+        # Brute-force fixing of final output header.
```

Comment:
Note that, the logic is nested here under the assumption that the header stuff will be fixed at HSTCAL eventually.